src/NHSD.BuyingCatalogue.SolutionLists.API/ViewModels/ListSolutionsFilterViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.SolutionLists.Contracts;
+
+namespace NHSD.BuyingCatalogue.SolutionLists.API.ViewModels
+{
+    /// <summary>
+    /// Provides the filter criteria for the <see cref="ListSolutionsQuery"/> query.
+    /// </summary>
+    public sealed class ListSolutionsFilterViewModel : IListSolutionsQueryData
+    {
+        public IEnumerable<CapabilityReferenceViewModel> Capabilities { get; } = new List<CapabilityReferenceViewModel>();
+
+        [JsonIgnore]
+        public IEnumerable<ICapabilityReference> CapabilityReferences
+        {
+            get
+            {
+                return Capabilities.OfType<ICapabilityReference>().ToList();
```

Comment:
Not sure we need the ToList & the OfType anymore here.

Reply:
👍 Brilliant catch! No need for that if we just return the Capabilities.